### PR TITLE
Feature/sl 871/transaction

### PR DESCRIPTION
**Pull request by radoslawlandowskikainos**

### CODE REVIEW

File: src/app/sessions/reducers/index.ts [hunk 1]
```diff
@@ -50,14 +50,22 @@ export const getSessionCreationEntitiesState = createSelector(
...
 );
 
-export const getRecentlyCreatedSessionId = createSelector(
+export const getRecentlyCreatedTransactionId = createSelector(
     getSessionCreationState,
-    fromSessionCreate.getRecent
+    fromSessionCreate.getRecent,
+);
+
+export const getRecentlyCreatedSessionId = createSelector(
```
Comment (leszekgu) [inline src/app/sessions/reducers/index.ts]: is this still used anywhere?

File: src/app/sessions/services/sessions-service.ts [hunk 2]
```diff
@@ -16,10 +16,16 @@ export class SessionsService {
     constructor(private http: HttpClient, private config: AppConfig) {
     }
 
+    getUserTransaction(id: string | String): Observable<any> {
```
Comment (leszekgu) [inline src/app/sessions/services/sessions-service.ts]: getSessionSByUserTransactionId ?

File: src/app/sessions/reducers/index.ts (hunk 1 above)
Comment (radoslawlandowskikainos) [reply src/app/sessions/reducers/index.ts]: It is in sessions-create.component. This id is used to filter the problems from state to display only the ones associated with the created session.

session created -> Get problems for session -> add problems to ProblemsState -> Get problems for session from ProblemsState

File: src/app/sessions/services/sessions-service.ts (hunk 2 above)
Comment (radoslawlandowskikainos) [reply src/app/sessions/services/sessions-service.ts]: This method gets only the user transaction. The casting to <Session> is misleading and should not be there.

**Pull request merged**